cli/src/internal/mcp/generator.go

```diff
@@ -195,16 +296,16 @@ func InitializeMCPServer(url string) (string, error) {
 	}
 	httpReq, err := http.NewRequest("POST", url, bytes.NewBuffer(data))
 	if err != nil {
-		fmt.Println("ERROR: Failed to create init request:", err)
-		return "", err
+		return "", fmt.Errorf("failed to create init request: %w", err)
 	}
 	httpReq.Header.Set("Content-Type", "application/json")
 	httpReq.Header.Set("Accept", "application/json, text/event-stream")
-	client := &http.Client{}
+	client := &http.Client{
+		Timeout: 10 * time.Second,
```

Comment:
Check values for the timeout

Reply:
https://github.com/wso2/api-platform/pull/472#discussion_r2622209947